app/models/event_log.rb

```diff
@@ -6,53 +6,55 @@ class EventLog < ActiveRecord::Base
-    UNSUCCESSFUL_LOGIN = LogEntry.new(id: 12, description: "Unsuccessful login"),
-    SUSPENDED_ACCOUNT_AUTHENTICATED_LOGIN = LogEntry.new(id: 13, description: "Unsuccessful login attempt to a suspended account, with the correct username and password"),
-    UNSUCCESSFUL_PASSWORD_CHANGE = LogEntry.new(id: 14, description: "Unsuccessful password change"),
-    EMAIL_CHANGED = LogEntry.new(id: 15, description: "Email changed", require_initiator: true),
-    EMAIL_CHANGE_INITIATED = LogEntry.new(id: 16, description: "Email change initiated"),
-    EMAIL_CHANGE_CONFIRMED = LogEntry.new(id: 17, description: "Email change confirmed"),
-    TWO_STEP_ENABLED = LogEntry.new(id: 18, description: "2-step verification enabled"),
-    TWO_STEP_RESET = LogEntry.new(id: 19, description: "2-step verification reset"),
-    TWO_STEP_ENABLE_FAILED = LogEntry.new(id: 20, description: "2-step verification setup failed"),
-    TWO_STEP_VERIFIED = LogEntry.new(id: 21, description: "2-step verification successful"),
-    TWO_STEP_VERIFICATION_FAILED = LogEntry.new(id: 22, description: "2-step verification failed"),
-    TWO_STEP_LOCKED = LogEntry.new(id: 23, description: "2-step verification failed too many times, account locked for #{LOCKED_DURATION}"),
-    TWO_STEP_CHANGED = LogEntry.new(id: 24, description: "2-step verification phone changed"),
-    TWO_STEP_CHANGE_FAILED = LogEntry.new(id: 25, description: "2-step verification phone change failed"),
-    TWO_STEP_PROMPT_DEFERRED = LogEntry.new(id: 26, description: "2-step prompt deferred"),
-    API_USER_CREATED = LogEntry.new(id: 27, description: "Account created", require_initiator: true),
-    ACCESS_TOKEN_REGENERATED = LogEntry.new(id: 28, description: "Access token re-generated", require_application: true),
-    ACCESS_TOKEN_GENERATED = LogEntry.new(id: 29, description: "Access token generated", require_application: true, require_initiator: true),
-    ACCESS_TOKEN_REVOKED = LogEntry.new(id: 30, description: "Access token revoked", require_application: true, require_initiator: true),
-    PASSWORD_RESET_LOADED_BUT_TOKEN_EXPIRED = LogEntry.new(id: 31, description: "Password reset page loaded but the token has expired"),
-    SUCCESSFUL_PASSWORD_RESET = LogEntry.new(id: 32, description: "Password reset successfully"),
-    ROLE_CHANGED = LogEntry.new(id: 33, description: "Role changed", require_initiator: true),
-    ACCOUNT_UPDATED = LogEntry.new(id: 34, description: "Account updated", require_initiator: true),
-    PERMISSIONS_ADDED = LogEntry.new(id: 35, description: "Permissions added", require_initiator: true),
-    PERMISSIONS_REMOVED = LogEntry.new(id: 36, description: "Permissions removed", require_initiator: true),
-    ACCOUNT_INVITED = LogEntry.new(id: 37, description: "Account was invited", require_initiator: true),
+    ACCOUNT_LOCKED = LogEntry.new(id: 1, description: "Password verification failed too many times, account locked for #{LOCKED_DURATION}", require_uid: true),
```

Comment:
I wonder if the default for `require_uid ` should be `true` as nearly all of these `LogEntry`s have `require_uid: true`.

Reply:
I thought about that, but it would be inconsistent with the others, which could be confusing

Reply:
Fair enough